sites/public/pages/listing/[id]/[slug].tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import Head from "next/head"
+import axios from "axios"
+import { Listing } from "@bloom-housing/backend-core/types"
+import { imageUrlFromListing, MetaTags, t } from "@bloom-housing/ui-components"
+import Layout from "../../../layouts/application"
+import { ListingView } from "../../../src/ListingView"
+
+interface ListingProps {
+  listing: Listing
+}
+
+export default function ListingPage(props: ListingProps) {
+  const { listing } = props
+  const pageTitle = `${listing.name} - ${t("nav.siteTitle")}`
+  const metaDescription = t("pageDescription.listing", {
+    regionName: t("region.name"),
+    listingName: listing.name,
+  })
+  const metaImage = imageUrlFromListing(listing)
+
+  return (
+    <Layout>
+      <Head>
+        <title>{pageTitle}</title>
+      </Head>
+      <MetaTags title={listing.name} image={metaImage} description={metaDescription} />
+      <ListingView listing={listing} />
+    </Layout>
+  )
+}
+
+export async function getStaticPaths(context: any) {
```

Comment:
I think `any` will produce additional lining warnings, so can we add proper types (preferred option) or disable `eslint@typescript-eslint/no-explicit-any` in these lines?